Runtime/Helpers/CinemachineDefaultMouseInput.cs

```diff
@@ -0,0 +1,75 @@
+#if CINEMACHINE_UNITY_INPUTSYSTEM
+
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace Cinemachine
+{
+    /// <summary>
+    /// Static class that returns a reference to InputSystem package's PlayerLook input asset.
+    /// </summary>
+    static class CinemachineDefaultMouseInput
+    {
+        static InputActionReference s_InputActionReference = null;
+        static CinemachineDefaultMouseInput()
+        {
+            var inputActionAsset = AssetDatabase.LoadAssetAtPath<InputActionAsset>("Packages/com.unity.inputsystem/" +
+                "InputSystem/Plugins/PlayerInput/DefaultInputActions.inputactions");
```

Comment:
I'm not wild about this as a static constructor.  I think an initialize-on-demand pattern would be better

Reply:
I agree. Was thinking about that yesterday too.